lib/design_system/onboarding/avatar_widget.dart

```diff
@@ -13,19 +13,19 @@ class AvatarWidget extends StatelessWidget {
     return Stack(
       alignment: Alignment.center,
       children: [
-        SizedBox(
```

Comment:
Esse sizedBox ta sem parênteses de fechamento, nao sei se o Dart chia com isso, but be careful.